Sources/JXKit/JXCancellable.swift

```diff
@@ -0,0 +1,17 @@
+/// Type similar to `Combine.Cancellable` that is used to cancel subscriptions to events.
+public class JXCancellable {
```

Comment:
I added our own simple Listener/Cancellable system. JXBridge also takes advantage of it. It minimizes our reliance on Combine. Also, I wasn't sure how to synchronously catch errors from Combine listeners, and in JXBridge I wanted to deal with listener errors.